rest/test/routes/transactionRoutes_spec.js

```diff
@@ -18,96 +18,275 @@
 const catapult = require('catapult-sdk');
+const { expect } = require('chai');
+const sinon = require('sinon');
 
+const { address } = catapult.model;
 const { convert } = catapult.utils;
 
 describe('transaction routes', () => {
-	describe('PUT transaction', () => {
-		test.route.packet.addPutPacketRouteTests(transactionRoutes.register, {
-			routeName: '/transaction',
-			packetType: '9',
-			inputs: {
-				valid: {
-					params: { payload: '123456' },
-					parsed: Buffer.of(
-						0x0B, 0x00, 0x00, 0x00, // size (header)
-						0x09, 0x00, 0x00, 0x00, // type (header)
-						0x12, 0x34, 0x56 // payload
-					)
-				},
-				invalid: {
-					params: { payload: '1234S6' },
-					error: { key: 'payload' }
+	describe('transaction', () => {
+		describe('PUT transaction', () => {
+			test.route.packet.addPutPacketRouteTests(transactionRoutes.register, {
+				routeName: '/transaction',
+				packetType: '9',
+				inputs: {
+					valid: {
+						params: { payload: '123456' },
+						parsed: Buffer.of(
+							0x0B, 0x00, 0x00, 0x00, // size (header)
+							0x09, 0x00, 0x00, 0x00, // type (header)
+							0x12, 0x34, 0x56 // payload
+						)
+					},
+					invalid: {
+						params: { payload: '1234S6' },
+						error: { key: 'payload' }
+					}
 				}
-			}
+			});
 		});
-	});
 
-	describe('get', () => {
-		const addGetPostTests = (dbApiName, key, ids, parsedIds) => {
-			const errorMessage = 'has an invalid format';
-			test.route.document.addGetPostDocumentRouteTests(transactionRoutes.register, {
-				routes: { singular: '/transaction/:transactionId', plural: '/transaction' },
-				inputs: {
-					valid: { object: { transactionId: ids[0] }, parsed: [parsedIds[0]], printable: ids[0] },
-					validMultiple: { object: { transactionIds: ids }, parsed: parsedIds },
-					invalid: { object: { transactionId: '12345' }, error: `transactionId ${errorMessage}` },
-					invalidMultiple: {
-						object: { transactionIds: ['12345', ids[0], ids[1]] },
-						error: `element in array transactionIds ${errorMessage}`
-					}
-				},
-				dbApiName,
-				type: 'transactionWithMetadata'
+		describe('get', () => {
+			const addGetPostTests = (dbApiName, key, ids, parsedIds) => {
+				const errorMessage = 'has an invalid format';
+				test.route.document.addGetPostDocumentRouteTests(transactionRoutes.register, {
+					routes: { singular: '/transaction/:transactionId', plural: '/transaction' },
+					inputs: {
+						valid: { object: { transactionId: ids[0] }, parsed: [parsedIds[0]], printable: ids[0] },
+						validMultiple: { object: { transactionIds: ids }, parsed: parsedIds },
+						invalid: { object: { transactionId: '12345' }, error: `transactionId ${errorMessage}` },
+						invalidMultiple: {
+							object: { transactionIds: ['12345', ids[0], ids[1]] },
+							error: `element in array transactionIds ${errorMessage}`
+						}
+					},
+					dbApiName,
+					type: 'transactionWithMetadata'
```

Comment:
i know this test already existed but this happens in the new test as well:

please **import** and use `routeResultTypes.transaction` instead